### ISSUE TITLE
Cannot use OTF fonts for guifont

### ISSUE THREAD

**Issue by JSamir** [akiyosi/goneovim#199]
Hi :)

I switched to goneovim and was wondering why the font looked a bit mushy.

After a bit of try and error with `:set guifont` I see that any otf font I try does not work, but ttf fonts do. I have tried this with `Hasklig` and `Source Code Pro` which do not work. `Hack` and `Liberation Mono` work.

Same happens if I use `:set guifont=*` and select the fonts.

**Comment by akiyosi** [comment]
Hi :D 
I have added font family detection logging in debug mode to the latest package.
Could you please DL the latest package and check what font families are actually selected when changing font families with `set guifont=` in debug mode?

https://github.com/akiyosi/goneovim/actions/runs/599846180

**Comment by JSamir** [comment]
Ok, I had to google how to download artifacts from the Github Actions page :-D
I do not know what the different linux builds are for, but I could not start version `5.15.1`:
```
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
```
Pretty sure I already had this problem some time ago but could not remember how to fix it. Anyways, I tried `5.13.0` next. Not only could I launch without errors, but also the font can be set with this binary. My other local installation still has the same issue though.

Is the info from the debug mode needed anymore?
